### PR TITLE
Add k8s example configs

### PR DESCRIPTION
**Pull request by tkyi**
Move sample configs here since kubernetes repo has been migrated

### CODE REVIEW

**Review by minzcmu — CHANGES_REQUESTED** [review]
License?

File: README.md [hunk 1]
```diff
@@ -1 +1,19 @@
...
+- [Artifact Store Docker image](https://hub.docker.com/r/screwdrivercd/store)
+
+### Usage
+
+Describe current status:
+```
+$ kubectl describe -f FILENAME
+```
+
+Update deployment:
```
Comment (minzcmu) [inline README.md]: This can also be used to create a new deployment. Will `Create or update` be better?

**Review by minzcmu — CHANGES_REQUESTED** [review]

File: kubernetes/api.yaml [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+          httpGet:
+            path: /v4/status
+            port: 80
+          initialDelaySeconds: 15
+          timeoutSeconds: 1
+        # Environment variables that point to your secrets in your secrets.yaml
+        env:
+          # You can set secrets directly using the format name/value
+          - name: DATASTORE_PLUGIN
+            value: dynamodb
```
Comment (d2lam) [inline kubernetes/api.yaml]: Change this value? this is not valid anymore.

File: kubernetes/api.yaml [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+            valueFrom:
+              secretKeyRef:
+                name: secrets
+                key: oauthclientsecret
+          - name: SECRET_PASSWORD
+            valueFrom:
+              secretKeyRef:
+                name: secrets
+                key: password
+          - name: DATASTORE_DYNAMODB_ID
```
Comment (d2lam) [inline kubernetes/api.yaml]: also this

**Review by d2lam — CHANGES_REQUESTED** [review]

**Review by d2lam — APPROVED** [review]
LGTM 👍

**Review by petey — APPROVED** [review]

**Review by minzcmu — APPROVED** [review]

**Pull request merged**